Controller/Store/GetStoredetails.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ *
+ * Copyright © 2013-2017 Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace RicardoMartins\PagSeguro\Controller\Store;
+
+use Magento\Framework\Controller\ResultFactory;
+//use RicardoMartins\PagSeguro\Helper\Data;
+
+use Magento\Customer\Model\Address\Config;
+use Magento\Directory\Model\CountryFactory;
+use Magento\Directory\Model\RegionFactory;
+use Magento\Framework\DataObject;
+
+class GetStoredetails extends \Magento\Framework\App\Action\Action
+{    
+     /**
+     * PagSeguro Helper
+     *
+     * @var RicardoMartins\PagSeguro\Helper\Data;
+     */ 
+    
+
+   protected $_storeManager;    
+
+
+    /**
+     * @var CountryFactory
+     */
+    private $countryFactory;
+
+    /**
+     * @var RegionFactory
+     */
+    private $regionFactory;
+    private $countryInformation;
+    private $scopeConfig;
+    /**
+     * @var Config
+     */
+    private $addressConfig;
+    public function __construct( 
+        \Magento\Store\Model\StoreManagerInterface $storeManager, 
+\Magento\Framework\App\Action\Context $context,
+\Magento\Directory\Api\CountryInformationAcquirerInterface $countryInformation,
+\Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
+
+    )
+    {        
+      
+        $this->_storeManager = $storeManager;   
+    
+	$this->countryInformation = $countryInformation;
+	$this->scopeConfig = $scopeConfig;
+
+        parent::__construct($context);
+    }
+
+    /**
+     * Retrieve formatted store address from config
+     *
+     * @param Store $store
+     * @return string
+     */
+    public function getFormattedAddress(Store $store)
+    {
+        /** @var \Magento\Customer\Block\Address\Renderer\DefaultRenderer $renderer */
+        $renderer = $this->addressConfig->getFormatByCode('html')->getRenderer();
+        return $renderer->renderArray($this->getStoreInformationObject($store)->getData());
+    }
+    
+    /**
+     * Get store identifier
+     *
+     * @return  int
+     */
+    public function getStoreId()
```

Comment:
we don't need this info. Exposing this could be seen by merchant's as unnecessary risk